src/components/Form.tsx

```diff
@@ -31,6 +35,16 @@ function Form<S extends InputSettings>(props: IFormProps<S>) {
   const [responsesChanged, setResponsesChanged] = useState(false);
   const inputs: ReactNode[] = [];
 
+  // Include a static warning message
+if (props.staticMessage) {
```

Comment:
Please indent this block at the same level as the surrounding code.

Reply:
Indent corrected in latest commit